source/manual/review-apps.html.md

```diff
@@ -0,0 +1,166 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-05-20
+title: Review apps for Pull Requests
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+# Review apps for Pull Requests
+
+Review apps allow us to automatically deploy application changes into a brand
```

Comment:
Might be worth linking to https://devcenter.heroku.com/articles/github-integration-review-apps